commands/user.go

```diff
@@ -221,6 +231,7 @@ func init() {
 	UserCreateCmd.Flags().String("lastname", "", "Optional. The last name for the new user account")
 	UserCreateCmd.Flags().String("locale", "", "Optional. The locale (ex: en, fr) for the new user account")
 	UserCreateCmd.Flags().Bool("system_admin", false, "Optional. If supplied, the new user will be a system administrator. Defaults to false")
+	UserCreateCmd.Flags().Bool("email_verified", false, "Optional. If supplied, the new user will have the email verified. Defaults to false")
```

Comment:
I think we are moving to dash approach for flags instead of underscores. But the other ones are with underscore so to keep it more consistent I understand this but I'm still inclined to use `-`. What do you think @mgdelacroix?

Reply:
I'm inclined too but we should do that in another PR and that could be a breaking change for users that have automated scripts with that way of calling. Also mixing dash and underscore is confusing :(

Reply:
I see, makes sense :)